contrib/ecl/template.py

```diff
@@ -0,0 +1,59 @@
+pkgname = "ecl"
+pkgver = "23.9.9"
+pkgrel = 0
+build_style = "configure"
```

Comment:
gnu_configure, and the configure.ac to reconf is present in ./src . not sure how to regen it in another folder though

Reply:
ah, i guess `configure_gen = ["autoreconf", "-if", "src"]`.

you'll also need `make_dir = "."` and `automake` in `hostmakedepends`